web/vtadmin/src/components/routes/Schemas.tsx

```diff
@@ -32,7 +33,11 @@ export const Schemas = () => {
             <tr key={idx}>
                 <td>{row.cluster?.name}</td>
                 <td>{row.keyspace}</td>
-                <td>{row.tableDefinition?.name}</td>
+                <td>
+                    <Link to={`/schema/${row.cluster?.id}/${row.keyspace}/${row.tableDefinition?.name}`}>
```

Comment:
Does this generate an invalid link if `row.cluster` and `row.tableDefinition` aren't defined?

Reply:
Yeahhhh... so this is one of the annoying things about all proto fields being optional, even when they basically never are in practice. Validating all the pieces _all the time_ can be so cumbersome.

In this case though, I was just being lazy. I've updated it so that in the rare instance that we _can't_ generate a link, the table name will be rendered as plain text (assuming it's defined... which, y'know, it almost certainly will be).